core/src/perspectives/PerspectiveProxy.ts

```
@@ -784,16 +812,18 @@ export class PerspectiveProxy {
+     * and won't be executed until the batch is committed.
      */
-    async removeSubject<T>(subjectClass: T, exprAddr: string) {
+    async removeSubject<T>(subjectClass: T, exprAddr: string, batchId?: string) {
         let className = await this.stringOrTemplateObjectToSubjectClassName(subjectClass)
         let result = await this.infer(`subject_class("${className}", C), destructor(C, Actions)`)
         if(!result.length) {
-            throw "No constructor found for given subject class: " + className
+            throw "No destructor found for given subject class: " + className
         }
 
         let actions = result.map(x => eval(x.Actions))
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)